ledger/participant-state/kvutils/src/test/lib/scala/com/daml/ledger/TestLoggers.scala

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger
```

Comment:
It is very odd to have definitions under `com.daml.ledger` from `kvutils`.
I'd suggest moving these either to `com.daml.logging` or to at least under `com.daml.ledger.participant.state.kvutils`.